### ISSUE TITLE
cron.php executes successfully, Nextcloud says cronjob hasn't run

### ISSUE THREAD

**Issue by JosefWN** [nextcloud/server#24927]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Configure web cron in the Nextcloud UI
2. In my case, create a Kubernetes job that runs a `wget` call to `https://mysite.com/cron.php`
3. Response body is `{"status":"success"}` but the Nextcloud UI says: _Some jobs didn’t execute since X hours ago._

### Expected behaviour
The cron has seemingly run successfully, so I should not see this message in the UI? Can't see any issues in the Nextcloud log either.

### Actual behaviour
I get a warning message in the UI on the cron settings under _Basic settings_.

### Server configuration

**Operating system:** Linux x86_64

**Web server:** apache via ingress-nginx

**Database:** Postgres 13

**PHP version:** 7.4.13

**Nextcloud version:** 20.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install

**Where did you install Nextcloud from:** Using the docker image `nextcloud:20.0.4-apache`

**Comment by kesselb** [comment]
See https://github.com/nextcloud/server/issues/18582 and https://github.com/nextcloud/server/pull/19329.

**tl;dr:** Cron may work but there are too many jobs in the backlog. Try to run the wget job more often or switch to system cron (https://github.com/nextcloud/docker/tree/master/.examples).

**Comment by JosefWN** [comment]
That did the trick, thanks @kesselb!

**Comment by rfilmyer** [comment]
For anyone else who finds this, I found a substantial backlog in `oc_jobs` when I queried the database:

```
 SELECT COUNT(*) FROM oc_jobs WHERE UNIX_TIMESTAMP() - last_run > 60 * 60 * 24 * 30

+----------+
| COUNT(*) |
+----------+
|      912 |
+----------+
1 row in set (0.001 sec)
```

To clear the backlog, I just ran a script to call `cron.php` 900 times. Since I'm on windows, I used this powershell script:

```
1..10 | % {
	$ProgressPreference = 'SilentlyContinue'
	$Response = Invoke-Webrequest "https://nextcloud.example.com/cron.php"
	write "$Response $_"
}
```

(replace 10 with as many jobs as necessary)

**Comment by r2evans** [comment]
For postgres users, in @rfilmyer's sql statement replace `UNIX_TIMESTAMP()` with `extract(epoch from now())`.
